modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -2373,6 +2373,26 @@ private void _testBindObjectDefinitions(
 			Assert.assertEquals(
 				objectRelationship.getDeletionType(),
 				ObjectRelationshipConstants.DELETION_TYPE_CASCADE);
+
+			if (objectRelationship.getObjectDefinitionId1() !=
+					rootObjectDefinitionId) {
+
+				ObjectDefinition objectDefinition1 =
+					_objectDefinitionLocalService.getObjectDefinition(
+						objectRelationship.getObjectDefinitionId1());
+
+				Assert.assertFalse(objectDefinition1.isPortlet());
+			}
+
+			if (objectRelationship.getObjectDefinitionId2() !=
+					rootObjectDefinitionId) {
+
+				ObjectDefinition objectDefinition2 =
+					_objectDefinitionLocalService.getObjectDefinition(
+						objectRelationship.getObjectDefinitionId2());
+
+				Assert.assertFalse(objectDefinition2.isPortlet());
+			}
```

Comment:
Hi @larissacribeiro create a private method to assert this, for example:

```java
_assertRootDescendantPortlet(long objectDefinitionId) {
  // get object defintion

  if (objectDefinition.isRootDescendant()) {
    Assert.assertFalse(objectDefinition.isPortlet());
  }
}
```

Thanks!

Reply:
sure, just did!